Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -185,6 +185,7 @@ defineSuite([
 
             expect(tileset._geometricError).toEqual(240.0);
             expect(tileset._root).toBeDefined();
+            expect(tileset._root.descendantsWithContent).toBeUndefined();
```

Comment:
The new code paths should be tested ok by the examples. The coverage is still at 100%.